src/_guides/language/effective-dart/usage.md

```diff
@@ -721,22 +720,32 @@ then. That always happens after all initializers are run regardless of where
 useful. In fact, [dartdevc]({{site.webdev}}/tools/dartdevc)
 *requires* that it appear last.
 
-<div class="good">
+{% comment %}
+Question for Kathy: I've added a "Dart 2.0 note" below.
+Do we need to keep the last sentence above?
+{% endcomment %}
+
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/usage_good.dart (super-first)"?>
 {% prettify dart %}
 View(Style style, List children)
     : _children = children,
-      super(style) {
+      super(style);
 {% endprettify %}
-</div>
 
-<div class="bad">
+{:.bad-style}
+<?code-excerpt "misc/lib/effective_dart/usage_bad.dart (super-first)"?>
 {% prettify dart %}
 View(Style style, List children)
-    : super(style),
-      _children = children {
+    : super(style), // Error in Dart 2.0
```

Comment:
Do we also want this comment in the code?

Reply:
I'm on the fence about this. I suppose people might miss the note, but it's not super clear from the comment what exactly causes the error.

Reply:
I was on the fence too. I'll drop the code comment.